Gaijinizer patch/Map155.txt

```diff
@@ -57,26 +57,28 @@ When it comes to that, the human being captured nowadays is
 
 ユウマ隊長がいなければ、救出部隊をだすところだった…
 > CONTEXT: Map155/events/13/pages/0/29/Dialogue
-Camazotz is a rare encounter. Most of the reported cases
-were not released. If there was no Yuuma captain, it was
-about to send out a rescue unit ...#MTLed
+It's rare to encounter Camazotz, most reported cases haven't
+been publicly released. If Captain Yuuma hadn't been here,
+we would've been about to send out a rescue unit...#Ron
 > END STRING
```

Comment:
Minor flow:
>_**It's rare to encounter Camazotz... most reported cases
haven't even been publicly released. If Captain Yuuma hadn't
been here, we would've probably sent out a rescue unit...**_